micro-ui/web/micro-ui-internals/packages/modules/core/src/components/TopBarSideBar/SideBar/StaticCitizenSideBar.js

```
@@ -27,14 +27,16 @@ import LogoutDialog from "../../Dialog/LogoutDialog";
-
 /* 
 Feature :: Citizen Webview sidebar
 */
 const Profile = ({ info, stateName, t }) => (
   <div className="profile-section">
     <div className="imageloader imageloader-loaded">
-      <img className="img-responsive img-circle img-Profile" src={defaultImage} />
+      <img
+        className="img-responsive img-circle img-Profile"
+        src={defaultImage}
+      />
```

Biome

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.

(lint/a11y/useAltText)